plugins/inputs/statsd/statsd.go

```diff
@@ -283,9 +285,11 @@ func (s *Statsd) udpListen() error {
 				log.Printf("ERROR READ: %s\n", err.Error())
 				continue
 			}
+			bufCopy := make([]byte, n)
```

Comment:
I hadn't thought about using a pool, I'll have to look into that.

I'm not totally clear though how the pool handles sized slices, since that's the important bit of this line. Would I have to get from the pool and then resize each time? That seems like it would be less efficient than leaving it to the compiler but I could try to write up some benchmarking.